rest/admin_api.go

```diff
@@ -375,6 +398,11 @@ func (h *handler) handlePutDbConfig() (err error) {
 					return nil, err
 				}
 
+				bucketDbConfig.Version, err = GenerateDatabaseConfigVersionID(bucketDbConfig.Version, &bucketDbConfig)
```

Comment:
Similarly, don't we need to prevent users from specfiying `Version` in the REST API body?

Reply:
Yeah granted, this should be done